ci/gpu/build.sh

```diff
@@ -44,7 +44,12 @@ nvidia-smi
 
 logger "Activate conda env..."
 source activate gdf
-conda install "cudf=${MINOR_VERSION}.*" "cudatoolkit=$CUDA_REL" "gdal>=3.0.2"
+conda install "cudf=${MINOR_VERSION}.*" "cudatoolkit=$CUDA_REL" \
+    "rapids-build-env=$MINOR_VERSION.*"
```

Comment:
What happens to the gdal dependency in this change?

Reply:
It's in `rapids-build-env`

Reply:
Great, where is it? How can I take a look?

Reply:
hey @thomcom. we keep the conda environments in this repo: https://github.com/rapidsai/integration/

`rapids-build-env` specifically is located here https://github.com/rapidsai/integration/blob/branch-0.15/conda/recipes/rapids-build-env/meta.yaml

Reply:
https://github.com/rapidsai/integration/blob/branch-0.15/conda/recipes/rapids-build-env/meta.yaml#L70